src/main/controllers/UserResultsController.ts

```diff
@@ -4,9 +4,11 @@ import { convertISODateTimeToUTCFormat, sortRoles } from '../utils/utils';
 import { validateEmail } from '../utils/validation';
 import { RootController } from './RootController';
 import { NO_USER_MATCHES_ERROR, TOO_MANY_USERS_ERROR } from '../utils/error';
+import autobind from 'autobind-decorator';
 
+@autobind
 export class UserResultsController extends RootController {
-  public async post(req: AuthedRequest, res: Response): Promise<void> {
+  public async post(req: AuthedRequest, res: Response) {
```

Comment:
This must be ok but sometimes I just don't understand those typescripts' types. Here you removed the return type "Promise<void>" but in private async render(...) in RootController, the method still return Promise<void>.....never mind it is confusing for me :)